### ISSUE TITLE
Likes: "Likes" label missing from screen options

### ISSUE THREAD

**Issue by melchoyce** [Automattic/jetpack#10382]
#### Steps to reproduce the issue

1. Go to `Posts` -> `All Posts`
2. Click on `Screen Options`
3. Look at the `Columns` row
4. Label for the Likes column is missing

#### What I expected
The column checkmark to have a label.

#### What happened instead

<img width="1567" alt="screen shot 2018-10-23 at 4 56 06 pm" src="https://user-images.githubusercontent.com/2846578/47390263-f1a7e580-d6e4-11e8-85ad-09c5c2991964.png">
